xml/NZZ_groundtruth/nzz_18590920_0_0_a1_p1_1.xml

```diff
@@ -10262,12 +10262,12 @@ Anstalt je länger je mehr auf eigene Füße zu stellen.</Unicode>
                 <Word id="w_w1aab1c31b2b3b1ac57" language="German" custom="readingOrder {index:5;}">
                     <Coords points="996,3497 1246,3497 1246,3528 996,3528"/>
                     <TextEquiv>
-                        <Unicode>Pierre-Francois-¬</Unicode>
+                        <Unicode>Pierre-François-¬</Unicode>
```

Comment:
The line break used in the ground truth is inconsistent. Ideally it should use `⸗`, but here either `-` (like after Pierre) or `¬` (the usual line break used by ABBYY / Transkribus) should be used. Which variant is preferred?

Reply:
I am not entirely sure what would be the correct solution as the hyphen does not actually come from hyphenation. @pstroe I think we need a guideline here and then also check the consistency over the rest of the transcriptions.
![image](https://user-images.githubusercontent.com/3041168/82416787-ecef1300-9a7a-11ea-8e52-9bf7798b7b0e.png)